src/packagers/JSConcatPackager.js

```diff
@@ -78,6 +78,25 @@ class JSConcatPackager extends Packager {
     return this.size;
   }
 
+  findExportAsset(asset, name) {
```

Comment:
might be a good idea to share some of this function with the similar one in concat.js. probably should ensure that they work the same way.

Reply:
fixed